src/main/markers/waveform.segments.js

```diff
@@ -330,6 +330,16 @@ define([
     return this.segments;
   };
 
+  WaveformSegments.prototype.getSegment = function(id) {
+    for ( var i = 0; i < this.segments.length; i++ ) {
```

Comment:
Style nitpick: although not in our eslint config, this project doesn't use spaces inside parentheses (we should add `"space-in-parens": ["error", "never"]` to `.eslintrc.js`).